### PR TITLE
Use material design icons for custom blocks 

### LINKED ISSUE #115: Replace images used for custom blocks/pullouts
Related to #114. The images from Bookdown are a bit 90s looking. Let's use font awesome images and create consistent usage definitions.

### PR DESCRIPTION
**Pull request by willingc**
Closes #115

### CODE REVIEW

**Comment by willingc** [comment]
We may wish to use 2x instead of 1x.

File: style.css [hunk 1]
```diff
@@ -12,23 +12,27 @@ pre {
...
+.rmdnote,
+.rmdtip,
+.rmdwarning {
   padding: 1em 1em 1em 4em;
   margin-bottom: 10px;
   background: #f5f5f5 5px center/3em no-repeat;
 }
 .rmdcaution {
-  background-image: url("images/caution.png");
+  background-image: url("images/logos/caution.png");
```
Comment (dsblank) [inline style.css]: These should be /icon/ not /logos/, right?

Comment (willingc) [reply style.css]: Good catch.

**Comment by dsblank** [comment]
Thanks, @willingc ! I think we need to all of the tables (Pro Tips, definitions, etc). Do you want to do them in this PR?

**Comment by dsblank** [comment]
Here is a preview of the icons:

screenshot from 2018-12-06 12-29-15

**Comment by willingc** [comment]
I think that we should leave the structure in this PR and make changes in other PRs. I will also probably do a separate style PR too.

**Comment by labarba** [comment]
What about the text boxes that present tools? (Nbviewer, Anaconda, etc.). These are formatted as tables in the markdown, so will need an icon?

**Comment by dsblank** [comment]
Thanks @willingc ! Let's see how this looks from gh-pages.

**Comment by dsblank** [comment]
Looks great from gh-pages (i.e., the main book link). I had to do a hard cache clear and reload, FYI.

screenshot from 2018-12-06 12-50-46

**Comment by dsblank** [comment]
PDF looks good too, but the epub will need something added to make it work.
PDF:
screenshot from 2018-12-06 12-52-58
EPUB:
screenshot from 2018-12-06 12-58-34

**Pull request merged**